src/response/redirect.ts

```diff
@@ -5,9 +5,11 @@ const redirect: ResponseHelper = (
   _,
   location,
   statusCode = 302,
+  options,
```

Comment:
Should set this to a default value of `{}` and the type is `InterfaceResponseOptions`